client/src/App.tsx

```diff
@@ -1,44 +1,25 @@
-import * as React from 'react';
-import { Routes, Route, Link } from "react-router-dom";
+import { Routes, Route } from "react-router-dom";
+import Landing from './landing/Landing';
+import Loading from './loading/Loading';
+import Review from './review/Review';
+import ThankYou from './thankyou/ThankYou';
+import Upload from "./upload/Upload";
+import Workflow from "./workflow/Workflow";
 
 function App() {
   return (
     <div className="App">
-      <h1>Hello world!</h1>
       <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path="about" element={<About />} />
+        <Route path="/" element={<Landing />} />
+        <Route path="/loading" element={<Loading />} />
+        <Route path="/review" element={<Review />} />
+        <Route path="/thankyou" element={<ThankYou />} />
+        <Route path="/upload" element={<Upload />} />
+        <Route path="/workflow" element={<Workflow />} />
       </Routes>
     </div>
   );
 }
```

Comment:
To globally set the base url for api endpoints, we could either access from a `.env` file after proper setup, or use this simple code snippet I invented that works in `App.tsx`.

```
// Make API Base URL
const hostname = window.location.hostname;
const port = window.location.port;

if (hostname === "localhost" || hostname === "127.0.0.1" || port === 3000) {
    axios.defaults.baseURL = `//${hostname}:3001`;
    console.log("Running on Localhost", axios.defaults.baseURL);
} else {
    axios.defaults.baseURL = `https://api.${hostname}`;
    console.log("Running on Production", axios.defaults.baseURL);
}
```

Generally when FE is running locally, all requests should go to port `3001`, otherwise go to `https://api.lifeline.techstartucalgry.com`.

Reply:
Can you please do this?

Reply:
For sure, which way do you prefer? I could set up for `.env` file approach; and you just need to paste the above to the file if you like the code snipper approach.

Reply:
Let's go with the snippet for now, unless you think `.env` approach is better